anaconda_build/stream_engine/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
 
 source:
     git_url: https://github.com/oceanobservatories/stream_engine
-    git_rev: v1.3.7
+    git_rev: v1.3.8
```

Comment:
1.3.8 doesn't appear to be a valid release yet.  Will the merge of this code be held off until that tag becomes available? Just saying, If it's merged before 1.3.8 is a valid tag this won't work.

Reply:
I will have this tag and the one for ingest_engine in place in time for this.